### ISSUE TITLE
最新的develop版本搭配paddle3.0beta，mask rtdert结果BUG

### ISSUE THREAD

**Issue by David-dotcom666** [PaddlePaddle/PaddleDetection#9051]
### 问题确认 Search before asking

- [X] 我已经查询[历史issue](https://github.com/PaddlePaddle/PaddleDetection/issues)，没有发现相似的bug。I have searched the [issues](https://github.com/PaddlePaddle/PaddleDetection/issues) and found no similar bug report.

### Bug组件 Bug Component

Inference

### Bug描述 Describe the Bug

没有修改任何参数，训练自己的数据集,预测的时候看起来结果有点奇怪，box和区域对应不上
![Brightnessbaicah_8](https://github.com/PaddlePaddle/PaddleDetection/assets/84842493/8b4017a5-6113-45fb-ae6e-30d491145e55)
![Rotatebaicah_8](https://github.com/PaddlePaddle/PaddleDetection/assets/84842493/df7a1b95-6e36-4c2c-a1a3-10e25f73349a)
![Noisebaicah_8](https://github.com/PaddlePaddle/PaddleDetection/assets/84842493/7f9da701-c429-4407-9213-fa903731b0d9)
![FlipTbbaicah_8](https://github.com/PaddlePaddle/PaddleDetection/assets/84842493/6a828e1d-3472-498f-91de-1ea243df93ed)
![Cropbaicah_8](https://github.com/PaddlePaddle/PaddleDetection/assets/84842493/fda8ad88-bf6f-4801-97cd-ac4e2bf2f78f)

### 复现环境 Environment

paddle3.0beta
paddledetection devlop

### Bug描述确认 Bug description confirmation

- [X] 我确认已经提供了Bug复现步骤、代码改动说明、以及环境信息，确认问题是可以复现的。I confirm that the bug replication steps, code change instructions, and environment information have been provided, and the problem can be reproduced.

### 是否愿意提交PR？ Are you willing to submit a PR?

- [ ] 我愿意提交PR！I'd like to help by submitting a PR!

**Comment by David-dotcom666** [comment]
..突然发现是不是检测和分割结果不一定是一致的原因？

**Comment by David-dotcom666** [comment]
歪下,今天测试另外一个数据集,在aistudio上测试，paddle3.0beta1不收敛了......很奇怪啊.数据集应该没问题，我其他模型也都有测试。loss试过0.01，0.001，0.0001等.都是一样
[07/16 18:44:33] ppdet.utils.checkpoint INFO: Save checkpoint: output/mask_rtdetr_hgnetv2_l_6x_coco
[07/16 18:44:35] ppdet.engine INFO: Epoch: [68] [ 0/27] learning_rate: 0.009181 loss_class: 0.367967 loss_bbox: 2.756441 loss_giou: 3.488019 loss_mask: 0.072027 loss_dice: 4.220462 loss_class_aux: 3.544642 loss_bbox_aux: 17.603238 loss_giou_aux: 21.985734 loss_mask_aux: 0.515226 loss_dice_aux: 29.176123 loss_class_dn: 0.265038 loss_bbox_dn: 0.271326 loss_giou_dn: 1.797755 loss_mask_dn: 0.073608 loss_dice_dn: 4.266407 loss_class_aux_dn: 4.928043 loss_bbox_aux_dn: 1.671240 loss_giou_aux_dn: 10.564847 loss_mask_aux_dn: 0.438327 loss_dice_aux_dn: 25.037251 loss: 134.331268 eta: 0:01:57 batch_cost: 1.1258 data_cost: 0.3871 ips: 3.5531 images/s max_mem_reserved: 20924 MB max_mem_allocated: 15603 MB
[07/16 18:45:10] ppdet.utils.checkpoint INFO: Save checkpoint: output/mask_rtdetr_hgnetv2_l_6x_coco
[07/16 18:45:13] ppdet.engine INFO: Epoch: [69] [ 0/27] learning_rate: 0.009316 loss_class: 0.353973 loss_bbox: 2.668644 loss_giou: 3.468810 loss_mask: 0.073804 loss_dice: 4.220462 loss_class_aux: 3.570071 loss_bbox_aux: 16.972439 loss_giou_aux: 21.772073 loss_mask_aux: 0.515226 loss_dice_aux: 29.323551 loss_class_dn: 0.254117 loss_bbox_dn: 0.270471 loss_giou_dn: 1.800367 loss_mask_dn: 0.075378 loss_dice_dn: 4.282680 loss_class_aux_dn: 4.960308 loss_bbox_aux_dn: 1.653066 loss_giou_aux_dn: 10.584920 loss_mask_aux_dn: 0.444510 loss_dice_aux_dn: 25.048894 loss: 134.015442 eta: 0:01:27 batch_cost: 1.1234 data_cost: 0.3754 ips: 3.5606 images/s max_mem_reserved: 20924 MB max_mem_allocated: 15603 MB
[07/16 18:45:55] ppdet.utils.checkpoint INFO: Save checkpoint: output/mask_rtdetr_hgnetv2_l_6x_coco
[07/16 18:45:56] ppdet.engine INFO: Epoch: [70] [ 0/27] learning_rate: 0.009451 loss_class: 0.379243 loss_bbox: 2.462223 loss_giou: 3.453747 loss_mask: 0.071520 loss_dice: 4.215566 loss_class_aux: 3.584089 loss_bbox_aux: 16.287401 loss_giou_aux: 21.733148 loss_mask_aux: 0.508052 loss_dice_aux: 29.242014 loss_class_dn: 0.272200 loss_bbox_dn: 0.273875 loss_giou_dn: 1.808867 loss_mask_dn: 0.069109 loss_dice_dn: 4.279628 loss_class_aux_dn: 5.003366 loss_bbox_aux_dn: 1.679549 loss_giou_aux_dn: 10.615354 loss_mask_aux_dn: 0.429924 loss_dice_aux_dn: 25.196416 loss: 133.948853 eta: 0:00:58 batch_cost: 1.1487 data_cost: 0.4132 ips: 3.4821 images/s max_mem_reserved: 20924 MB max_mem_allocated: 15603 MB
[07/16 18:46:37] ppdet.utils.checkpoint INFO: Save checkpoint: output/mask_rtdetr_hgnetv2_l_6x_coco
[07/16 18:46:38] ppdet.engine INFO: Epoch: [71] [ 0/27] learning_rate: 0.009585 loss_class: 0.373000 loss_bbox: 2.158872 loss_giou: 3.401268 loss_mask: 0.073804 loss_dice: 4.223697 loss_class_aux: 3.773967 loss_bbox_aux: 14.960081 loss_giou_aux: 21.540277 loss_mask_aux: 0.515226 loss_dice_aux: 29.414947 loss_class_dn: 0.307338 loss_bbox_dn: 0.276058 loss_giou_dn: 1.810359 loss_mask_dn: 0.069611 loss_dice_dn: 4.302322 loss_class_aux_dn: 5.013556 loss_bbox_aux_dn: 1.687848 loss_giou_aux_dn: 10.616708 loss_mask_aux_dn: 0.444510 loss_dice_aux_dn: 25.415340 loss: 133.159714 eta: 0:00:29 batch_cost: 1.1922 data_cost: 0.4556 ips: 3.3550 images/s max_mem_reserved: 20924 MB max_mem_allocated: 15603 MB
[07/16 18:47:16] ppdet.utils.checkpoint INFO: Save checkpoint: output/mask_rtdetr_hgnetv2_l_6x_coco
我本地训练的(paddle3.0.0b0)也是，不同的数据集
07/16 18:58:39] ppdet.engine INFO: Epoch: [55] [  0/147] learning_rate: 0.000100 loss_class: 1.605693 loss_bbox: 0.061972 loss_giou: 0.301805 loss_mask: 0.028925 loss_dice: 0.835592 loss_class_aux: 12.156880 loss_bbox_aux: 0.686644 lo
ss_giou_aux: 2.883258 loss_mask_aux: 0.204407 loss_dice_aux: 5.983186 loss_class_dn: 1.477559 loss_bbox_dn: 0.077520 loss_giou_dn: 0.354221 loss_mask_dn: 0.035275 loss_dice_dn: 0.911075 loss_class_aux_dn: 7.915819 loss_bbox_aux_dn: 1.1
94533 loss_giou_aux_dn: 4.297459 loss_mask_aux_dn: 0.301137 loss_dice_aux_dn: 7.775783 loss: 50.629913 eta: 0:40:52 batch_cost: 0.9317 data_cost: 0.0015 ips: 2.1467 images/s max_mem_reserved: 9886 MB max_mem_allocated: 8246 MB
[07/16 19:01:15] ppdet.utils.checkpoint INFO: Save checkpoint: output\mask_rtdetr_hgnetv2_l_6x_coco
[07/16 19:01:16] ppdet.engine INFO: Epoch: [56] [  0/147] learning_rate: 0.000100 loss_class: 1.609814 loss_bbox: 0.058103 loss_giou: 0.293592 loss_mask: 0.027431 loss_dice: 0.850178 loss_class_aux: 12.166869 loss_bbox_aux: 0.619530 lo
ss_giou_aux: 2.686366 loss_mask_aux: 0.202075 loss_dice_aux: 5.858680 loss_class_dn: 1.469288 loss_bbox_dn: 0.069964 loss_giou_dn: 0.345674 loss_mask_dn: 0.034220 loss_dice_dn: 0.911346 loss_class_aux_dn: 7.877977 loss_bbox_aux_dn: 1.1
65454 loss_giou_aux_dn: 4.205096 loss_mask_aux_dn: 0.285142 loss_dice_aux_dn: 7.676756 loss: 50.028254 eta: 0:38:28 batch_cost: 0.9725 data_cost: 0.0017 ips: 2.0566 images/s max_mem_reserved: 9886 MB max_mem_allocated: 8246 MB
[07/16 19:03:54] ppdet.utils.checkpoint INFO: Save checkpoint: output\mask_rtdetr_hgnetv2_l_6x_coco
[07/16 19:03:55] ppdet.engine INFO: Epoch: [57] [  0/147] learning_rate: 0.000100 loss_class: 1.590434 loss_bbox: 0.060139 loss_giou: 0.275657 loss_mask: 0.026760 loss_dice: 0.810847 loss_class_aux: 12.184828 loss_bbox_aux: 0.567063 lo
ss_giou_aux: 2.588926 loss_mask_aux: 0.196995 loss_dice_aux: 5.811783 loss_class_dn: 1.487064 loss_bbox_dn: 0.069206 loss_giou_dn: 0.339213 loss_mask_dn: 0.032761 loss_dice_dn: 0.893281 loss_class_aux_dn: 7.908192 loss_bbox_aux_dn: 1.1
38402 loss_giou_aux_dn: 4.152448 loss_mask_aux_dn: 0.273751 loss_dice_aux_dn: 7.592636 loss: 49.426710 eta: 0:36:05 batch_cost: 1.0017 data_cost: 0.0018 ips: 1.9965 images/s max_mem_reserved: 9886 MB max_mem_allocated: 8246 MB
可以提供数据集看看能不能查看到什么问题

**Comment by David-dotcom666** [comment]
..有什么回复或者建议，解释之类的吗？今天做了其他数据集的测试,也是会出现这种情况
![0UPTA-670M53K732240614P5000163_00_light1_r1_ch_1_2_3_4_r2_ch_1_2_3_4_5_6_7_8_r3_ch_1_2_3_4_5_6_7_8_9_10_11_12_13_14_15_16_r4_ch_1_2_3_4_us165_2024_06_14-08_11_30_537 (2)](https://github.com/user-attachments/assets/4e991a97-e4a2-4ec1-9dc6-2e9b9bb6810a)

**Comment by yski** [comment]
应该是你代码绘图的的问题，我这测试没有这个问题。我遇到的问题是trt开不起来
